Doc/build-en.md

```diff
@@ -99,6 +100,10 @@ Make sure all other components are installed before adding switches.
 Place the plate with keyholes over top the PCB and snap in and solder switches.
 Start with 4 corners in the plate to ensure alignment.
 After the first 4 are soldered in, install remaining switches.
+Just above the "ergoDash Rev 1.2" sign, there is thow possible location for the switch.
```

Comment:
Typo/grammar: "there is thow possible location" should be "there _are_ _two_ possible _locations_"